libsolidity/parsing/Token.cpp

```diff
@@ -155,16 +155,15 @@ tuple<Token::Value, unsigned int, unsigned int> Token::fromIdentifierOrKeyword(s
 			) {
 				int n = parseSize(positionX + 1, _literal.end());
 				if (
-					0 < m && m < 256 &&
-					0 < n && n < 256 &&
+					m + n > 0 &&
```

Comment:
Ok, all I want you to test is that it is not `-1`, because that is the error case.